docs/en/ufs/CephFS.md

```diff
@@ -0,0 +1,192 @@
+---
+layout: global
+title: CephFS
+nickname: CephFS
+group: Storage Integrations
+priority: 11
+---
+
+* Table of Contents
+{:toc}
+
+This guide describes how to configure Alluxio with CephFS as the under storage system. Alluxio supports
+two different under storage systems to connect to [CephFS](https://docs.ceph.com/en/latest/cephfs/):
+- [cephfs](https://docs.ceph.com/en/latest/cephfs/api/libcephfs-java/)
+- [cephfs-hadoop](https://docs.ceph.com/en/nautilus/cephfs/hadoop/)
+
+## Prerequisites
+
+### Deploy Alluxio binary package
+The Alluxio binaries must be on your machine. You can either
+[compile Alluxio]({{ '/en/contributor/Building-Alluxio-From-Source.html' | relativize_url }}), or
+[download the binaries locally]({{ '/en/deploy/Running-Alluxio-Locally.html' | relativize_url }}).
+
+### Install Dependences
+According to [ceph packages install](https://docs.ceph.com/en/latest/install/get-packages/)，
+    cephfs-java
+    libcephfs_jni
+    libcephfs2
+
+### Make symbolic links
+
+```console
+$ ln -s /usr/lib64/libcephfs_jni.so.1.0.0 /usr/lib64/libcephfs_jni.so
+$ ln -s /usr/lib64/libcephfs.so.2.0.0 /usr/lib64/libcephfs.so
+$ java_path=`which java | xargs readlink | sed 's#/bin/java##g'`
+$ ln -s /usr/share/java/libcephfs.jar $java_path/jre/lib/ext/libcephfs.jar
+```
+
+### Download CephFS Hadoop jar
+
+```console
+$ curl -o $java_path/jre/lib/ext/hadoop-cephfs.jar -s https://download.ceph.com/tarballs/hadoop-cephfs.jar
+```
+
+## Basic Setup
+
+Configure Alluxio to use under storage systems by modifying
+`conf/alluxio-site.properties` and `conf/core-site.xml`. If them do not exist, 
+create the configuration files from the templates
+
+```console
+$ cp conf/alluxio-site.properties.template conf/alluxio-site.properties
+$ cp conf/core-site.xml.template conf/core-site.xml
+```
+
+### Option 1: ufs cephfs
```

Comment:
checkout the new way of doing this to implement tabs:

```
{% navtabs Setup %}
{% navtab Option 1: cephfs %}
....
{% endnavtab %}
{% navtab Option 2: cephfs-hadoop %}
...
{% endnavtab %}
{% endnavtabs %}
```

See how it looks like at [this page](https://docs.alluxio.io/os/user/stable/en/deploy/Running-Alluxio-On-Docker.html#launch-alluxio-containers-for-master-and-worker)